src/test/java/io/jenkins/plugins/analysis/core/scm/GitBlamerITest.java

```diff
@@ -39,6 +38,14 @@
  */
 @SuppressWarnings("PMD.SignatureDeclareThrowsException")
 public class GitBlamerITest extends IntegrationTestWithJenkinsPerTest {
+    private static final String DETAILS = "Details";
+    private static final String FILE = "File";
+    private static final String AGE = "Age";
+    private static final String AUTHOR = "Author";
+    private static final String EMAIL = "Email"; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field EMAIL has the same name as a method](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)